### ISSUE TITLE
PUMI 2.2.6

### ISSUE THREAD

**Issue by cwsmith** [SCOREC/core#341]
This issue is to document functionality and features added to PUMI 2.2.6 since the [2.2.5 release](https://github.com/SCOREC/core/issues/326):

- Improved high order mesh adaptation and field support (#331, #327, #311)                                                                                                                                                                                                                                                     
- Simmetrix SimModSuite support increased to 15.0-210501 (993ee2f)                                                                                                                                                                                                                                                             
- GCC 11 support (#339)                                                                                                                                                                                                                                                                                                        
- GCC 10 bugfix (#334)

**Comment by cwsmith** [comment]
@mortezah Do you see any commits in the list below that should be highlighted or any edits to the feature list in the opening post?

```
X b8b3681 (origin/develop, develop) found was not null terminated, use strcpy - #334                                                                                                                                                                                                                                         
X 993ee2f simmodsuite max version to 15.0-210501                                                                                                                                                                                                                                                                             
X 06119fc Merge branch 'cws/gcc11-fixes' into develop - #339                                                                                                                                                                                                                                                                 
15bb927 (seb/develop) Adds a new functionality to maDBG - improves mesh adapt debugging capabilities                                                                                                                                                                                                                         
x 1848358 test requires parasolid - fix broken ctest logic                                                                                                                                                                                                                                                                   
4d73368 fix overflow                                                                                                                                                                                                                                                                                                         
X aa71ef0 Merge pull request #335 from jacobmerson/fix-issue-334                                                                                                                                                                                                                                                             
6ae8ffd Merging develop into master                                                                                                                                                                                                                                                                                          
X 439d9a8 fixes issue #334                                                                                                                                                                                                                                                                                                   
X 684c0f6 Merge pull request #331 from avinmoharana/updateBezierMesh - API for converting interpolating points to Bezier control points                                                                                                                                                                                      
a7b5a01 Adds visualizeSizeField to maDBG                                                                                                                                                                                                                                                                                     
18cdb53 Merging develop into master                                                                                                                                                                                                                                                                                          
X 65b33bb Merge pull request #311 from Samiullah-Malik/error_estimation                                                                                                                                                                                                                                                      
7b22d3f adds assert for mesh dimension                                                                                                                                                                                                                                                                                       
cc64814 removes ternary statement and replaces tabs with spaces                                                                                                                                                                                                                                                              
a6c6a7d Merge branch 'error_estimation' of https://github.com/Samiullah-Malik/core into error_estimation                                                                                                                                                                                                                     
b4426c6 further clean ups                                                                                                                                                                                                                                                                                                    
b6d2d75 clean ups                                                                                                                                                                                                                                                                                                            
c826c75 Updates the pumi-meshes submodule                                                                                                                                                                                                                                                                                    
a0f05d1 Reverts "Adds residual based error estimation test" partially                                                                                                                                                                                                                                                        
547d012 Removes the remaining dependence on crv                                                                                                                                                                                                                                                                              
12b10c0 Merge branch 'develop' into error_estimation                                                                                                                                                                                                                                                                         
0c48da0 Replaces couts with lion_oprint                                                                                                                                                                                                                                                                                      
4020c21 Renames the residual test for consistency                                                                                                                                                                                                                                                                            
ddd720b Removed the dependence on crv                                                                                                                                                                                                                                                                                        
9279eea Removed unneeded header                                                                                                                                                                                                                                                                                              
091f578 Small style changes to ree.h                                                                                                                                                                                                                                                                                         
d9f3bea updates residual test to use 1k edge coarse mesh                                                                                                                                                                                                                                                                     
aae6841 cleans ups in ree subdirectory                                                                                                                                                                                                                                                                                       
0eab738 clean up of 2d piola transformation                                                                                                                                                                                                                                                                                  
ecbdeda changes getElementDofs to getElementNodeData                                                                                                                                                                                                                                                                         
1738518 updates mesh and model path for ree test                                                                                                                                                                                                                                                                             
595d2da fixes and add checks in residualErrorEstimation test                                                                                                                                                                                                                                                                 
d56aff1 Deletes em subdirectory                                                                                                                                                                                                                                                                                              
b17b9cb changes CMake em to ree                                                                                                                                                                                                                                                                                              
0a70462 Delete .nfs000000000002a1270000bfa3                                                                                                                                                                                                                                                                                  
5422a1c Delete .nfs00000000000286a50000bdda                                                                                                                                                                                                                                                                                  
18afbd6 Changes the name of the subdirectory 'em' to 'ree'.                                                                                                                                                                                                                                                                  
e8e74f2 Cleans up the error estimation files.                                                                                                                                                                                                                                                                                
6f2a3a3 Adds residualErrorEstimation test in testing.cmake                                                                                                                                                                                                                                                                   
7bc29a3 Adds residual based error estimation test                                                                                                                                                                                                                                                                            
b31683d Merging develop into master                                                                                                                                                                                                                                                                                          
X 8521d59 Merge pull request #327 from avinmoharana/higherOrderSizeField - Implementation of higher order size fields for mesh adapt and the corresponding tests                                                                                                                                                             
ace1b97 citing pumi                                                                                                                                                                                                                                                                                                          
4d10723 Fixes typo                                                                                                                                                                                                                                                                                                           
473e074 Removes extra fields info from HO sizefield test                                                                                                                                                                                                                                                                     
8e013bc Adds the test for higher order size fields                                                                                                                                                                                                                                                                           
c19dba2 Adds member initializations for the default c-tors                                                                                                                                                                                                                                                                   
af885e3 Updates the SizeFieldIntegrator                                                                                                                                                                                                                                                                                      
c4f914e Cleans up ma::SizeField class                                                                                                                                                                                                                                                                                        
7a4f450 using order 3 integrator for sizeField->measure                                                                                                                                                                                                                                                                      
7e5aa0b updates the transfer in FaceSplit to account for SizeFiled.                                                                                                                                                                                                                                                          
d9bec9f Merging develop into master                                                                                                                                                                                                                                                                                          
80c3e1c Removes commented code                                                                                                                                                                                                                                                                                               
927bebd Merge branch 'develop' into higherOrderSizeField                                                                                                                                                                                                                                                                     
b2928b8 Merging develop into master                                                                                                                                                                                                                                                                                          
f77fee2 Moves the c-tor/d-tor for MatrixElement to apfMatrixElement.cc                                                                                                                                                                                                                                                       
08f4a31 Fixes a bug in getElementData                                                                                                                                                                                                                                                                                        
5898aa3 Removes the obsolete comment                                                                                                                                                                                                                                                                                         
1b30311 Updates H1Shape test to test the matrix fields too                                                                                                                                                                                                                                                                   
659aa23 Merging develop into master                                                                                                                                                                                                                                                                                          
6cea22b Exposes cruved_vtk_writers for python                                                                                                                                                                                                                                                                                
d8da034 Merging develop into master                                                                                                                                                                                                                                                                                          
a540535 Exposes curve adapt options in python wrappers                                                                                                                                                                                                                                                                       
e460dba removed declaration of sizeField in Cavity class + implements hasNodesOn for sizeField.                                                                                                                                                                                                                              
8d322a7 Merging develop into master                                                                                                                                                                                                                                                                                          
9544912 Adds a few new apis to python_wrappers                                                                                                                                                                                                                                                                               
ae0952c Nedelec fields can now be projected to IPShapes                                                                                                                                                                                                                                                                      
166b1d0 Updates projectNedelecField                                                                                                                                                                                                                                                                                          
8f2ee41 Adds minor changes to H1 shapes.                                                                                                                                                                                                                                                                                     
78bb490 Merging develop into master   
```

**Comment by mortezah** [comment]
@cwsmith the current list looks complete.
